purchase_supplierinfo_product_breakdown/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
```

Comment:
* In Product, enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.